app/models/batch/tecan_behaviour.rb

```diff
@@ -17,14 +20,15 @@ def generate_tecan_data(target_barcode, override_plate_type = nil)
       full_source_barcode = request.asset.plate.machine_barcode
       full_destination_barcode = request.target_asset.plate.machine_barcode
 
-      source_plate_name = override_plate_type.presence || request.asset.plate.plate_type.tr('_', "\s")
-
```

Comment:
Move this inside the conditional that determines if we actually make use of it. (Most of this was happening while I was trying to track down why things were so slow)

Reply:
Also, renamed it to source_plate_type, because that's what it is.